MonoGame.Framework/Quaternion.cs

```diff
@@ -748,10 +748,10 @@ public static Quaternion Slerp(Quaternion quaternion1, Quaternion quaternion2, f
 		    }
 		    else
 		    {
-		        float num5 = (float) Math.Acos((double) num4);
-		        float num6 = (float) (1.0 / Math.Sin((double) num5));
```

Comment:
Precision lost!
The division (reciprocal) was calculated with double precision.